lib/pagy.rb

```diff
@@ -34,6 +34,7 @@ def initialize(vars)
 
   # Return the array of page numbers and :gap items e.g. [1, :gap, 7, 8, "9", 10, 11, :gap, 36]
   def series(size=@vars[:size])
+    return [] if size.empty?
```

Comment:
Infinite-extra does not have information does not perform COUNT query. That's why shoring "series" is redundant.
This line make all Frontend render only "Prev | Next" links.

It works for my current project. And it allows easy switching between standard backend and infinite "per-controller".

Reply:
I actually wanted to do it. But `series` methods belongs to `Pagy` itself. And I can't figure out hot wo override it.

Reply:
Just define it like:
```rb
class Pagy
  def series(...)
    ...
  end

  module Backend
    ...
  end
end
```